addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -168,7 +168,7 @@ registerModel({
          * @returns {boolean}
          */
         _computeIsDisabled() {
-            return Boolean(!this.thread || this.thread.isTemporary);
+            return Boolean(!this.thread || this.thread.isTemporary || (this.thread && !this.thread.hasAccess));
```

Comment:
no need for extra condition on thread, if thread is not defined it will return false with the first condition

Reply:
> no need for extra condition on thread, if thread is not defined it will return false with the first condition


Done.Thanks